src/vstart.sh

```diff
@@ -247,6 +247,7 @@ options:
 	--jaeger: use jaegertracing for tracing
 	--seastore-devs: comma-separated list of blockdevs to use for seastore
         --seastore-secondary-des: comma-separated list of secondary blockdevs to use for seastore
```

Comment:
Outline in GitHub does look shifted. Not clear is that is due to `tab` versus `space`.
But I would expect it to align??

Reply:
yeah, that line has spaces (others has tab). @nmshelke care to quickly fix it?

Reply:
Done.